docfx_project/ziti/quickstarts/zac/installation.md

```diff
@@ -123,43 +123,43 @@ this time.  It's not difficult to reuse the PKI but you'll need to do the follow
 4. Now add the ZAC configuration lines to the compose file of your choice:
    ```bash
      ziti-console:
-    image: openziti/zac
-    ports:
-      - "1408:1408"
-      - "8443:8443"
-    environment:
-      - ZITI_EDGE_CONTROLLER_HOSTNAME=ziti-edge-router
-    volumes:
-      - ziti-fs:/openziti
-      - type: bind
-        source: ./ziti-edge-controller-server.key
-        target: /usr/src/app/server.key
-      - type: bind
-        source: ./ziti-edge-controller-server.chain.pem
-        target: /usr/src/app/server.chain.pem
-
-    networks:
-      - zitiblue
-      - zitired
+       image: openziti/zac
+       ports:
+         - "1408:1408"
+         - "8443:8443"
+       environment:
+         - ZITI_EDGE_CONTROLLER_HOSTNAME=ziti-edge-router
+       volumes:
+         - ziti-fs:/openziti
+         - type: bind
+           source: ./ziti-edge-controller-server.key
+           target: /usr/src/app/server.key
+         - type: bind
+           source: ./ziti-edge-controller-server.chain.pem
+           target: /usr/src/app/server.chain.pem
+
+       networks:
+         - zitiblue
+         - zitired
    ```
 1. After adding the ZAC configuration as shown, `docker-compose` will now start and expose the ZAC ports on 1408/8443.
-   > [!Note]
-   > Do note that if you are exposing ports as shown above, you will need to ensure that `ziti-edge-controller` is
-   > addressable by your machine in order to use docker in this way. This guide does not go into how to do this in depth.
-   > One easy, and common mechanism to do this would be to edit the 'hosts' file of your operating system. A quick
-   > internet search should show you how to accomplish this.
-   > 
+> [!Note]

```

Comment:
I'm not sure if this was intended but this note is an exact copy of the note just before this section. They have different commit timestamps but I'm assuming that, based on the fact that they are identical and not just closely worded, that this was intended so I'm leaving it in for now.